
package com.custodix.mpi._1_0;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.5.2
 * 2014-01-27T11:59:07.057+01:00
 * Generated source version: 2.5.2
 */

@WebFault(name = "InternalError", targetNamespace = "http://www.custodix.com/MPI/1.0")
public class InternalError extends Exception {
    
    private com.custodix.mpi._1.InternalError internalError;

    public InternalError() {
        super();
    }
    
    public InternalError(String message) {
        super(message);
    }
    
    public InternalError(String message, Throwable cause) {
        super(message, cause);
    }

    public InternalError(String message, com.custodix.mpi._1.InternalError internalError) {
        super(message);
        this.internalError = internalError;
    }

    public InternalError(String message, com.custodix.mpi._1.InternalError internalError, Throwable cause) {
        super(message, cause);
        this.internalError = internalError;
    }

    public com.custodix.mpi._1.InternalError getFaultInfo() {
        return this.internalError;
    }
}
